README.md

```diff
@@ -37,3 +37,31 @@ The relayer consists of the following components:
 <div align="center">
   <img src="resources/relayer-diagram.png?raw=true">
 </div>
+
+## Testing
+
+---
+
+### Unit tests
+
+Unit tests can be ran locally by running the command in root of the project:
+
+```bash
+./scripts/test.sh
+```
+
+### E2E tests
```

Comment:
Can you add a note here to install Gingko following the instructions [here](https://onsi.github.io/ginkgo/#installing-ginkgo)?